connectors/opa/main.go

```diff
@@ -17,10 +17,6 @@ import (
 
 const (
 	envOPAServerURL             = "OPA_SERVER_URL"
-	envCatalogConnectorURL      = "CATALOG_CONNECTOR_URL"
-	envCatalogProviderName      = "CATALOG_PROVIDER_NAME"
-	envConnectionTimeout        = "CONNECTION_TIMEOUT"
-	envDefaultConnectionTimeout = 10
```

Comment:
When you remove these lines, redundant spaces appear causing `verify` to fail. Please re-format.

Reply:
Fixed. Thanks